### PR TITLE
fix(type): update v11 token values

### LINKED ISSUE #11865: Update v11 type tokens in monorepo
**Checklist**

- [ ] Update root token values with the full change list for v11 in the [parent issue](https://github.com/carbon-design-system/carbon/issues/11813)

### PR DESCRIPTION
**Pull request by tay1orjones**
Closes #11865 

#### Changelog

**New**
- add token detail collapsible section to type elements example preview

**Changed**

- update type token values as specified in #11813 
- update the type elements example preview to use v11

#### Testing / Reviewing

* View the elements type preview, ensure the values have been updated to the new values

### CODE REVIEW

**Review by tw15egan — APPROVED** [review]
LGTM 👍 ✅

File: packages/type/scss/_styles.scss [hunk 1]
```diff
@@ -338,17 +339,17 @@ $fluid-heading-04: $expressive-heading-04 !default;
...
   letter-spacing: 0,
   breakpoints: (
     md: (
       font-size: scale.type-scale(9),
-      font-weight: font-family.font-weight('light'),
       line-height: 1.22,
     ),
     lg: (
       font-size: scale.type-scale(10),
+      font-weight: font-family.font-weight('regular'),
```
Comment (aagonzales) [inline packages/type/scss/_styles.scss]: `expressive-heading-04` at `lg` breakpoint should have a `light` weight.

Comment (aagonzales) [reply packages/type/scss/_styles.scss]: Or am i reading it wrong, hard to tell sometimes

File: packages/type/src/styles.js [hunk 2]
```diff
@@ -202,31 +203,28 @@ export const expressiveHeading04 = {
...
     md: {
       fontSize: rem(scale[8]),
-      fontWeight: fontWeights.light,
       lineHeight: 1.22,
       letterSpacing: 0,
     },
     lg: {
       fontSize: rem(scale[9]),
-      fontWeight: fontWeights.light,
+      fontWeight: fontWeights.regular,
```
Comment (aagonzales) [inline packages/type/src/styles.js]: I think this one stays `light`

**Review by aagonzales — COMMENTED** [review]
I think the weight is wrong in two spots, but I might be reading something wrong too.

**Review by aagonzales — APPROVED** [review]

**Pull request merged**